docs/doxygen/meson.build

```diff
@@ -15,32 +15,30 @@ footerfile = meson.current_source_dir() / 'html/footerFile.html'
 cssfile = meson.current_source_dir() / 'html/styleSheetFile.css'
 examples = meson.project_source_root() / 'samples'
 
-conf_data = configuration_data({
-    'DOXYGEN_INPUT': ' '.join(input_files),
-    'DOXYGEN_LAYOUT': layout,
-    'DOXYGEN_LOGO': logo,
-    'DOXYGEN_HEADER': headerfile,
-    'DOXYGEN_FOOTER': footerfile,
-    'DOXYGEN_CSS': cssfile,
-    'DOXYGEN_EXAMPLES': examples,
-    'DOXYGEN_OUTPUT': join_paths(meson.current_build_dir(), 'api'),
-    'DOXYGEN_WARN_AS_ERROR': get_option('werror') ? 'YES' : 'NO',
-})
-
 doxyfile = configure_file(
     input: 'doxyfile.in',
     output: 'doxyfile',
-    configuration: conf_data
+    configuration: configuration_data({
+        'DOXYGEN_INPUT': ' '.join(input_files),
+        'DOXYGEN_LAYOUT': layout,
+        'DOXYGEN_LOGO': logo,
+        'DOXYGEN_HEADER': headerfile,
+        'DOXYGEN_FOOTER': footerfile,
+        'DOXYGEN_CSS': cssfile,
+        'DOXYGEN_EXAMPLES': examples,
+        'DOXYGEN_OUTPUT': meson.current_build_dir() / 'output',
+        'DOXYGEN_WARN_AS_ERROR': get_option('werror') ? 'YES' : 'NO',
+    })
 )
 
-doxygen = custom_target(
-    'doxygen',
+docs = custom_target(
+    'docs',
     build_by_default: true,
     input: doxyfile,
+    output: 'output',
     command: [
-        sh,
```

Comment:
Does anyone care that I reverted this?

Reply:
Describe the impact of what you reverted.

Reply:
... cause I can't tell :)

Reply:
Essentially all the stdout from doxygen was getting sent to /dev/null. Now it will just go to your terminal.

Reply:
This might need changes to the [github.io docs ](https://github.com/hse-project/hse-project.github.io/blob/master/generate-api.sh#L38)

Reply:
Bhavesh, I am working on a major overhaul for that. I will ping you when the PR is up.